content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -223,18 +246,48 @@ plus a few additional polyfills:
   - `browserify-zlib`
   - `path-browserify`
   - `react-native-url-polyfill`
+  - `react-native-quick-crypto`
+
+```bash
+  # Using npm
+  npm install @metaplex-foundation/js@0.19.4 \
+            assert \
+            util \
+            crypto-browserify \
+            stream-browserify \
+            readable-stream \
+            browserify-zlib \
+            path-browserify \
+            react-native-url-polyfill \
+            react-native-quick-crypto
+  # Using yarn
+  yarn add @metaplex-foundation/js@0.19.4 \
+         assert \
+         util \
+         crypto-browserify \
+         stream-browserify \
+         readable-stream \
+         browserify-zlib \
+         path-browserify \
+         react-native-url-polyfill \
+         react-native-quick-crypto
+```
 
 All of the libraries that the above polyfills are meant to replace are utilized
 by the Metaplex library in the background. It's unlikely you'll be importing any
 of them into your code directly. Because of this, you'll need to register the
-polyfills using a `metro.config.js` file. This will ensure that Metaplex uses
-the polyfills instead of the usual Node.js libraries that aren't supported in
-React Native. Below is an example `metro.config.js` file:
+polyfills using a `metro.config.js` and `babel.config.js` file. This will ensure
+that Metaplex uses the polyfills instead of the usual Node.js libraries that
+aren't supported in React Native. Below is an example `metro.config.js` file:
```

Comment:
This may not be necessary as the metaplex client changes (it's now `mpl-token-metadata`)/

Reply:
the babel config changes will not be needed but probably the mpl-token-metadata too uses node api's internally so some polyfills will still be needed